mkdocs.yml

```diff
@@ -8793,9 +8794,9 @@ nav:
 plugins:
 - tags
 - search
-- git-revision-date-localized:
-    enable_creation_date: true
-    type: timeago
+#- git-revision-date-localized:
```

Comment:
Any reason for removing date stamps on the docs ?

Reply:
Pushed new change.